source/fundamentals/linq.txt

```diff
@@ -0,0 +1,531 @@
+   ``AsQueryable`` method.
+
+   To learn how to make a collection queryable, see :ref:`csharp-linq-queryable`.
+
+Consider a second collection in the ``sample_restaurants`` database called
+``reviews`` that has restaurant reviews. The following ``Reviews`` class
+models the documents in that collection:
+
+.. literalinclude:: /includes/fundamentals/code-examples/linq.cs
+   :language: csharp
+   :dedent:
+   :start-after: start-review-model
+   :end-before: end-review-model
+
+Select the :guilabel:`Query Syntax` or :guilabel:`Method Syntax` tab to see how 
+how to generate an ``$lookup`` stage using LINQ:
+
+.. tabs::
+
+   .. tab:: Query Syntax
+      :tabid: query-syntax
+
+      .. code-block:: csharp
+
+         var query = from restaurant in queryableCollection
+             join rv in reviewCollection on restaurant.Name equals rv.RestaurantName into reviews
+             select new { restaurant, reviews };
+
+   .. tab:: Method Syntax
+      :tabid: method-syntax
+
+      .. code-block:: csharp
+
+         var query = queryableCollection
+             .GroupJoin(reviewCollection,
+                 restaurant => restaurant.Name,
+                 review => review.RestaurantName,
+                 (restaurant, reviews) =>
+                     new {restaurant, reviews}
+             );
+
+The preceding example returns documents from the ``restaurants`` collection, each
+with an added field called ``reviews``. The added field contains all documents from the
+``reviews`` collection with a ``name`` field that matches the ``name`` of the 
+document from the ``restaurants`` collection. 
+
+The following is a subset of the returned results:
+
+.. code-block:: json
+
+   // Results Truncated
+
+   { "restaurant" : {
+      "_id" : ObjectId("..."),
+      "name" : "The Movable Feast", 
+      "restaurant_id" : "40361606", 
+      "cuisine" : "American", 
+      "address" : {...}, 
+      "borough" : "Brooklyn", 
+      "grades" : [...] }, 
+      "reviews" : [
+        { "_id" : ObjectId(...), "restaurant_name" : "The Movable Feast", "reviewer" : "Lazlo Cravensworth", "review" : "Great restaurant! 12/10 stars!" },
+        { "_id" : ObjectId("..."), "restaurant_name" : "The Movable Feast", "reviewer" : "Michael Scarn", "review" : "It really was a feast" }
+       ]
+    }
+
+Unsupported Aggregation Stages
+------------------------------
+
+The {+driver-long+} implementation of LINQ does not support the following
+aggregation stages:
+
+- ``$redact``
+- ``$geoNear``
+- ``$out``
+
+Supported Methods
+-----------------
+
+The {+driver-long+} implementation of LINQ supports the following methods:
+
+.. list-table::
+   :header-rows: 1
+   :widths: 40 60
+
+   * - Method Name
+     - Description
+   
+   * - ``Any``
+     - Determines if any documents match the specified criteria
+
+   * - ``Average``
+     - Calculates the average of the specified fields
+
+   * - ``Count``
+     - Returns the number of documents that match the specified criteria
```

Comment:
If ``LongCount`` is an Int64, presumably this is an Int32? Maybe include just for completeness.